respy/solve.py

```diff
@@ -63,7 +63,25 @@ def solve(params, options, state_space):
 
 @parallelize_across_dense_dimensions
 def _create_choice_rewards(complex_, choice_set, optim_paras, options):
-    """Create wage and non-pecuniary reward for each state and choice."""
+    """Create wage and non-pecuniary reward for each state and choice.
+
+    In particular the function retrieves dense period choice cores
+    with all covariates (they have aready been calculated in the
+    construction of the state space) from disk.
+    Thereafter the function obtains rewards for choices for each
+    state based on the pre calculated covariates.
+
+    Returns
+    ----------
+    wages : np.array
+        Array with dimensions n_states x n_choices.
+        Contains all wages for a particular state choice
+        combination within a dense period choice core.
+    nonpecs : np.array
```

Comment:
```suggestion
    nonpecs : numpy.ndarray
```
This is the correct type.

Reply:
And you need to write out numpy and pandas